### DISCUSSION TITLE
NotyKT v1.1.0

### DISCUSSION THREAD

**Discussion by PatilShreyas** [PatilShreyas/NotyKT#200] [General]
This release includes User experience improvements in the Jetpack Compose Application. No change in simple app.

### 🔮 What's New?**

- [[#117](https://github.com/PatilShreyas/NotyKT/issues/117)] Added screen: **About** for the details regarding application.
- [[#118](https://github.com/PatilShreyas/NotyKT/issues/118)] Added ***Swipe to refresh*** support in Notes screen to re-load the notes.

### ✅ Bug Fixes / Improvements**

All below fixes and improvements are done in the Compose application.

- [[#117](https://github.com/PatilShreyas/NotyKT/issues/117)] Clear all previous screens from backstack after successful login/signup.
- [[#117](https://github.com/PatilShreyas/NotyKT/issues/117)] Added validation for input text fields in Login and Signup screen.
- [[#120](https://github.com/PatilShreyas/NotyKT/issues/120)] Fix Background of Login screen in Dark mode (_Earlier, it's not supporting dark theme well_)
- [[#151](https://github.com/PatilShreyas/NotyKT/issues/117)] Added transition while navigating through the screens.
- [[#196](https://github.com/PatilShreyas/NotyKT/issues/197)] Avoid re-syncing notes every time whenever notes screen is launched
(_after returning to notes screen from other screens like About or note details_).
- [[#197](https://github.com/PatilShreyas/NotyKT/issues/197)] Improved UI/UX of the input Text fields throughout the application.
- Fix: Back button pressed in note details screen creates new Notes screen instead of going back.

### 🎯 Codebase Improvements**

- [[#117](https://github.com/PatilShreyas/NotyKT/issues/117)] Create re-usable Composable components to reduce the repetitive code.
- Migrated to the latest version of Dagger 2.31.2.
- Using the stable release of Jetpack Compose 1.0.1 with Kotlin 1.5.21.
- Renamed color components of theme.
- Use Hilt Compose navigation.
- Cleaned up code, refactored classes and composable methods.

<hr /><em>This discussion was created from the release <a href='https://github.com/PatilShreyas/NotyKT/releases/tag/v1.1.0'>NotyKT v1.1.0</a>.</em>

**Comment by deepakkumar-mahindra** [comment]
HI @PatilShreyas  I am currently rearchitecturing our android app. I followed this project for some best practices. Here is my one doubt. How edittext restore its text in loginfragment while configuration changes? I am not able to understand this behaviour.
